dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -358,10 +358,14 @@ $small-footer-standard-width: 400px;
   position: absolute;
   bottom: 0;
   left: 0;
-  width: 625px;
+  width: 650px;
```

Comment:
I thought it was nicer when the last bit about Code.org copyrights fit on a single line even when the vertical scroll bar is present.

`© Code.org, 2015. Code.org®, the CODE logo and Hour of Code® are trademarks of Code.org.`

Note, this is still bounded by the `max-width: 50%` and `min-width: 300px` rules below.